static/src/stylesheets/module/_weather.scss

```diff
@@ -104,7 +104,7 @@ $weather-large-size: 60px;
     }
 }
 
-.i.weather__icon {
+.weather__icon svg {
```

Comment:
fair enough.

i didn't want that span, it was a performance issue in scala (having to parse the svg vs stuff the output wrapped in a span). I'd like to find a way to remove it.